src/main/resources/userEmailTemplate-feedbackSessionClosed.html

```diff
@@ -47,10 +47,8 @@
 </p>
 
 <p>
-    <strong>No action is required if you have already submitted.</strong>
-    <br>Any request for deadline extensions should be sent to the instructors of the course,
-    not to the TEAMMATES support team. If you need any help or clarifications regarding TEAMMATES,
-    please email us at ${supportEmail}.
+    <strong>No action is required if you have already submitted. Any request for deadline extensions should be sent to the instructors of the course,</strong> not to the TEAMMATES support team.
```

Comment:
@leeyimin  Ok. So I should just keep it as it was while using the strong tag to define the sentence in question?